src/Common/PlanetParameters/PlanetaryConstants.jl

```diff
@@ -0,0 +1,105 @@
+"""
+    PlanetaryConstants
+
+Module containing physical constants and parameters characterizing the planet.
+"""
+module PlanetaryConstants
+
+using ..UniversalConstants
+
+export PlanetConstants,
+       AbstractPlanetConstants,
+       EarthConstants
+       # MarsConstants
+
+struct PlanetConstants{FT}
+  "Von Karman constant (1)"
+  k_Karman::FT
+  "Molecular weight dry air (kg/mol)"
+  molmass_dryair::FT
+  "Gas constant dry air (J/kg/K)"
+  R_d::FT
+  "Adiabatic exponent dry air"
+  kappa_d::FT
+  "Isobaric specific heat dry air"
+  cp_d::FT
+  "Isochoric specific heat dry air"
+  cv_d::FT
+  "Density of liquid water (kg/m^3)"
+  ρ_cloud_liq::FT
+  "Density of ice water (kg/m^3)"
+  ρ_cloud_ice::FT
```

Comment:
```suggestion
  ρ_ice::FT
```
(and as for liquid, then introduce `ρ_cloud_ice = ρ_ice`)